guides/hack/21-XHP/16-extending-examples/required-attributes.php.hhconfig

```diff
@@ -0,0 +1,4 @@
+disable_xhp_element_mangling=true
+enable_xhp_class_modifier=true
+disable_xhp_children_declarations=true
+check_xhp_attribute = true
```

Comment:
perhaps we should add support for a prepended/default hhconfig for examples in the test

Reply:
Doesn't really matter if we wait until these are on by default, I can just remove that commit then.